ci/plugins/mzcompose/hooks/pre-exit

```diff
@@ -38,6 +60,18 @@ buildkite-agent artifact upload ps-aux.log
 docker ps -a --no-trunc > docker-ps-a.log
 buildkite-agent artifact upload docker-ps-a.log
 
+if grep "error: missing record 0: Record {" services.log; then
+  known_issue 16778
+fi
+if grep "thread 'coordinator' panicked at 'can't persist timestamp: Error { kind: Stash(StashError { inner: Other(\"unexpected lower\") }) }'" services.log; then
```

Comment:
Regexps should ideally be as simple as possible, so that they continue to match in case of unrelated changes to the output. So here, `unexpected lower` would have been enough.